.github/workflows/llvm-project-tests.yml

```diff
@@ -58,6 +58,10 @@ jobs:
   lit-tests:
     name: Lit Tests
     runs-on: ${{ matrix.os }}
+    container:
+      image: ${{(startsWith(matrix.os, 'ubuntu') && format('ghcr.io/{0}/ci-ubuntu-22.04:latest', github.repository_owner)) || null}}
```

Comment:
Why is this set to pull from the repository owner rather than just the container in upstream? I would think we would always want to use the container for upstream, even for testing to prevent having to run the (now eight hour) job on your fork to upload the package there.